lib/test/requirements_resolver.rb

```diff
@@ -33,22 +35,35 @@ def dependency_map
         Test::Tasks::RunVitest => Test::Tasks::PnpmInstall,
         Test::Tasks::RunAnnotate => Test::Tasks::SetupDb,
         Test::Tasks::RunTypelizer => Test::Tasks::BuildFixtures,
-        Test::Tasks::ConvertSchemasToTs => nil,
+        Test::Tasks::ConvertSchemasToTs => Test::Tasks::SetupDb,
```

Comment:
(Unrelated change, but I happened to notice that (I think) `ConvertSchemasToTs` ultimately goes through Rails, which will error if the DB doesn't exist yet.)